### PR TITLE
docs: clarify the default sort order for both cli and bot

### PR DESCRIPTION
**Pull request by atuttle**
Closes: #368

**What**: Improves clarity of the default sorting of contributors. I also made the description more consistent between the two documents.

**Why**: If you have to make a choice, it's helpful to understand all available options.

**Checklist**:
- [x] Documentation
- [x] Ready to be merged
- [x] Added myself to contributors table.

### CODE REVIEW

**Comment by atuttle** [comment]
@all-contributors add @atuttle for docs

**Comment by Berkmann18** [comment]
> For the record, in case it should be changed to prevent similar issues again in the future, the title of the PR is the default from GitHub, a copy of the one commit message included in the PR. The commit message was generated by `npm run commit` as suggested by [the contributing doc](https://allcontributors.org/docs/en/project/contribute#committing-to-an-all-contributors-repo).
> 
> If the leading emoji is somehow problematic, perhaps that tool should be updated?

I've never had a commit comes out like this when using the Angular convention (which hasn't changed since the last time I used it).
It's not a problem per se for us but `conventional-changelog` won't be able to understand the commit since it's not following the convention set on the repo (I might be wrong tho).

**Comment by Berkmann18** [comment]
@atuttle Have you had a chance to look into my review comment?

File: docs/cli/configuration.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ These are the keys you can specify:
...
 | `contributorsPerLine`            | Maximum number of columns for the contributors table.                                               | Default: `7`                                                                                                |
-| `contributorsSortAlphabetically` | Sort the contributors table in alphabetical order. Must be either `true` or `false`.                | Default: `false`                                                                                                |
+| `contributorsSortAlphabetically` | Sort the contributors table in alphabetical order. When `false`, unsorted aka insert-order. (either `true` or `false`)                | Default: `false`                                                                                                |
```
[the remark this answers is not in the document]
Comment (Berkmann18) [reply docs/cli/configuration.md:26]: Friendly ping @atuttle

**Comment by atuttle** [comment]
I don't understand what you're asking me to do. I did as the instructions told me to: I ran `npm run commit` and it generated the commit message. I didn't specify any type of convention, I just used the defaults.

If there's something I can do to update this PR to help it get merged, please let me know what that would be, specifically.

File: docs/cli/configuration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (atuttle) [reply docs/cli/configuration.md:26]: I just saw this comment. I can try to reword it soon.

**Review by Berkmann18 — APPROVED** [review]
Awesome, thank you.

**Pull request merged**